nifi-extension-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-hikari-dbcp-service/pom.xml

```diff
@@ -53,5 +53,29 @@
             <artifactId>nifi-dbcp-utils</artifactId>
             <version>2.8.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.derby</groupId>
+            <artifactId>derby</artifactId>
```

Comment:
With the Apache Derby project now retired, it would be best to avoid introducing additional dependencies on the library. I am evaluating other options.

Reply:
Yes this make sense.  I was keeping it consistent with the DBCP tests that are making use of Apache Derby within the same bundle.  I've used H2 for similar test usecases in the past.  Not sure if that is under consideration.  https://www.h2database.com/html/license.html

Reply:
I am evaluating a migration to [HSQLDB](https://hsqldb.org/) for testing purposes, and it is used in several other Apache projects. I plan to have a pull request ready next week.